Resources/Prototypes/GameRules/meteorswarms.yml

```diff
@@ -4,6 +4,7 @@
   components:
   - type: GameRule
     minPlayers: 25
+    cancelPresetOnTooFewPlayers: false
   - type: MeteorScheduler
```

Comment:
wont this mean if only 24 people of say 200 ready up, there will never be any meteors

Reply:
this is true of literally any gamemode

Reply:
this is a scheduler its like not enough people being readied up so no basic station event scheduler